translations/es.ts

```diff
@@ -2142,7 +2142,7 @@ Será instalada cuando reinicies qTox.</translation>
     <message>
         <location filename="../src/widget/form/removefrienddialog.ui" line="39"/>
         <source>YES</source>
-        <translation>SI</translation>
+        <translation>SÍ</translation>
```

Comment:
yes, I know what tecleo is, was just aksing if the user really has to type it in, as I have not yet encountered that in my daily use of qtox. 
But you answered that alwready.
Since en mayúsculas you can omit la tilde, I will make use of this option then.